modules/ui.styl

```diff
@@ -167,7 +168,7 @@ formElementBase() // This could be a class that is extended rather than a mixin
     color: var(--color-white-soft)
     border-color: var(--color-red)
     &:hover
-      color: var(--color-red)
+      color: var(--color-black)
```

Comment:
This does look like a bug. But important to check what expected behaviour is for this button on the Figma designs as I think it might be red text with white background hover state that is expected.